desktop/cmp/panel/PanelModel.ts

```diff
@@ -352,6 +333,22 @@ export class PanelModel extends HoistModel {
         this.resizeWhileDragging = v;
     }
 
+    //---------------------------------------------
+    // Persistable Interface
+    //---------------------------------------------
+    getPersistableState(): PersistableState<PersistablePanelState> {
+        const ret: PersistablePanelState = {};
+        if (this.collapsible) ret.collapsed = this.collapsed;
+        if (this.resizable) ret.size = this.size;
```

Comment:
I suppose here is where we could look to convert the current pixel size back into a percentage based size, if defaultSize spec'd as a percentage...

Reply:
I think we're actually ok - the issue I was seeing was actually due to the percent-specified default size not complying with a specified minSize.